dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/DataProcessingStep.java

```diff
@@ -38,6 +38,7 @@ public interface DataProcessingStep extends RestProcessingStep {
     public static final String CCLICENSE_STEP_OPERATION_ENTRY = "cclicense/uri";
     public static final String ACCESS_CONDITION_STEP_OPERATION_ENTRY = "discoverable";
     public static final String ACCESS_CONDITION_POLICY_STEP_OPERATION_ENTRY = "accessConditions";
+    public static final String SHERPA_RETRIEVAL_TIME = "retrievalTime";
```

Comment:
Minor thing, but why is this constant in `DataProcessingStep` instead of being in `SherpaPolicyStep` (which is the one class where this constant is used).  It seems out of place here, especially since the constant is specific to Sherpa while `DataProcessingStep` is obviously generic.